flow-typed/npm/tone_v14.x.x.js

```diff
@@ -0,0 +1,82 @@
+declare module "tone" {
+    // TODO: There are two things called "Frequency" in Tone.js
```

Comment:
Now that we have renamed the type below to "FrequencyType", we can remove the "TODO" from the comment and keep a note about using the name "FrequencyType" for the "Frequency" type and the name "Frequency" for the function.

Reply:
I made a go at rewording this.